### DISCUSSION TITLE
Feature request: An email should be sent to an organization's registered email when an error accurs for a cloud site

### DISCUSSION THREAD

**Discussion by kristofferwebmind** [umbraco/Umbraco.Cloud.Issues#808] [Ideas]
### What
When an error occurs for an Umbraco cloud project, an email should be sent automatically to the registered Umbraco cloud profile email address.

### Why
There are tools like uptime robot that we can use to get alerted when a site goes down. However, when upgrades are applied automatically by Umbraco Cloud, it happens that those upgrades fail. When that happens, it does _not_ necessarily mean that the site goes down, so uptime robot will not send an alert. The error message is shown on the detail page for a cloud project. For an organization like us, that has around 65 cloud projects running, it takes an unreasonably large amount of time to click through every project to see whether an upgrade has been successful or if it has produced an error. An automatic email sent on the occasion of an error seems like a reasonable feature that would both save us time and make us more comfortable regarding opting-in (or not opting-out...) for automatic minor upgrades.

### Extra bonus request
The registered profile email address is not necessarily the email address used by the organization for those kind of emails. A field where we could store the specific email address to which we would like to have the alerts sent would be a super nice extra bonus feature.

**Comment by sajumb** [comment]
Hi @kristofferwebmind,
Thank you for your detailed feedback and suggestion regarding automatic email alerts for errors during Umbraco Cloud project upgrades. We understand how crucial it is to promptly identify and address any issues, especially in environments managing multiple projects like yours.

Your suggestion is under consideration, and while we cannot commit to a development timeline at this moment, we assure you that the concept of improving error visibility and communication within Umbraco Cloud is a priority for us. We will keep your proposal in mind as we continue to evolve Umbraco Cloud and will provide updates should we move forward with development in this area.

**Comment by pijemcolu** [comment]
Hi. Thanks for opening the issue.

I'll close this in favour of a consolidated alerting discussion: https://github.com/umbraco/Umbraco.Cloud.Issues/discussions/1042
